### ISSUE TITLE
processesId picker only show part of process on the remote machine

### ISSUE THREAD

**Issue by CatsOfMrXue** [microsoft/vscode-cpptools#10379]
### Environment

- OS and version:
Windows 10 profession 21H1
- VS Code:
Version: 1.74.2 (user setup)
Commit: e8a3071ea4344d9d48ef8a4df2c097372b0c5161
Date: 2022-12-20T10:29:14.590Z
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.19043
Sandboxed: No
- C/C++ extension:
Version 1.13.9
- OS and version of remote machine (if applicable):
Linux version 3.10.0-1160.71.1.el7.x86_64 (mockbuild@kbuilder.bsys.centos.org) (gcc version 4.8.5 20150623 (Red Hat 4.8.5-44) (GCC) ) #1 SMP Tue Jun 28 15:37:28 UTC 2022
- GDB / LLDB version:
GNU gdb (GDB) Red Hat Enterprise Linux 7.6.1-120.el7

### Bug Summary and Steps to Reproduce

Bug Summary:

I want to attch to the remote target with ExtendedRemote mode，so I configrate my launch.json like 
`{
            "name": "Attach with GDB",
            "type": "cppdbg",
            "request": "attach",
            "program": "<path to program>",
            "MIMode": "gdb",
            "miDebuggerPath": "/usr/bin/gdb",
            "miDebuggerServerAddress": "172.20.0.97:2159",
            "useExtendedRemote": true
 }`
on the remote machine, i use the command `gdbserver --multi :2159`
then,the ProcessId Picker show the processes on my remote machine,but it only show part of the process on remote machine,because i can't find the process that i want to debug.but if i add `"processId":"79068"` to the launch.json,it works fine.
the `79068` is the target process ID which i get from `ps -aux`.BTW,i can use `info os processes` to find the target process ID on remote machine.

Steps to reproduce:
1. type command `gdbserver --multi :2159` on remote machine
2. config the launch.json above
3. start debug with the configration named "Attach with GDB"
4. ProcessId Picker show only part of processes on the remote machine

### Debugger Configurations

```shell
{
            "name": "Attach with GDB",
            "type": "cppdbg",
            "request": "attach",
            "program": "<path to program>",
            "MIMode": "gdb",
            "miDebuggerPath": "/usr/bin/gdb",
            "miDebuggerServerAddress": "172.20.0.97:2159",
            "useExtendedRemote": true
 }
```

### Debugger Logs

```shell
there seem no infomation generated from the Debug Console or Output windows before i choose the exact process ID
```

**Comment by WardenGnaw** [comment]
@xisui-MSFT Can you help out on this issue of remote process picker not showing all of the process on the remote machine?

**Comment by xisui-MSFT** [comment]
@CatsOfMrXue Can you share the output of `info os processes`?

**Comment by CatsOfMrXue** [comment]
> @CatsOfMrXue Can you share the output of `info os processes`?
![屏幕截图 2023-01-18 111359](https://user-images.githubusercontent.com/46380455/213074558-bc009d51-f866-4a1b-82e5-9e1e6feb6568.png)
the processes that i get from part of `info os processes` output is i want to attach,they are daemon

**Comment by sean-mcmanus** [comment]
A fix is available with 1.14.1 (pre-release): https://github.com/microsoft/vscode-cpptools/releases/tag/v1.14.1
